README.md

```diff
@@ -13,6 +13,12 @@ This action sets by node environment for use in actions by:
 
 See [action.yml](action.yml)
 
+You can use any valid [SemVer](https://semver.org/) range supported by the [sem-ver](https://www.npmjs.com/package/semver#ranges) module.
+
+For example [`node-version: '*'`](https://www.npmjs.com/package/semver#x-ranges-12x-1x-12-) would satisfy the latest version.
+
```

Comment:
can point here for more examples: https://github.com/actions/setup-node/blob/main/.github/workflows/versions.yml